HW2/Sqrt.java

```diff
@@ -0,0 +1,17 @@
+package HW2;
+
+public class Sqrt {
+    public static void main(String[] args) {
+        mySqrt(4);
+    }
+
+    public static int mySqrt(int x) {
+        int result = 0;
+        if (x == 0) {
+            return 0;
+        } else {
+            result = (int) Math.sqrt(x);
```

Comment:
Math.sqrt is built-in function 
<img width="523" alt="image" src="https://user-images.githubusercontent.com/115036936/227879703-198e4fee-ec37-473d-baf7-49398f5facbd.png">